doc/modules/cross_validation.rst

```diff
@@ -251,6 +251,15 @@ Note that the result of this computation may be slightly different from those
 obtained using :func:`cross_val_score` as the elements are grouped in different
 ways.
 
+Note on inappropriate usage of cross_val_predict
+================================================
+The function :func:`cross_val_score` takes an average of over cross-validation
+folds and grouped in different ways on computation in comparison to :func:`cross_val_predict`.
+Thus, any metric score on :func:`cross_val_predict` is not equivalent to the
+result of :func:`cross_val_score`. The function :func:`cross_val_predict` should
+only be used to evaluate a classifier and is not an appropriate measure of
```

Comment:
I think I'd should *not* be used to evaluate a classifier, but perhaps to do some diagnostic analysis or visualisation of predictions...

Reply:
Oh, In that case, a small query. In the original user guide it suggests so -- 
at http://scikit-learn.org/stable/modules/cross_validation.html
<img width="815" alt="screen shot 2017-10-08 at 10 03 46 am" src="https://user-images.githubusercontent.com/24438869/31313942-16504432-ac10-11e7-9707-a341592af4d3.png">
Shall I remove that line of phrase or just in the notes?

Reply:
"These prediction can then be used to evaluate the classifier:"